src/components/calcite-panel/calcite-panel.scss

```diff
@@ -96,10 +96,8 @@
   .summary {
     @apply p-0
       block
-      overflow-hidden
-      whitespace-no-wrap
-      w-full;
-    text-overflow: ellipsis;
+      pointer-events-none
```

Comment:
why no pointer events here?

Reply:
oh! You're right. Thanks for catching that